packages/1password/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.32.0"
+  changes:
+    - description: Update Kibana constraint to support 9.0.0.
```

```suggestion
    - description: Add Kibana constraint to support 9.0.0.
```

Its better to have this changelog, since we are not updating, but adding 9.0.0

---

The kibana constraint is being modified to add the 9.0.0 semver clause. I think update matches this.